pkg/apis/serving/v1/configuration_validation_test.go

```diff
@@ -384,18 +384,6 @@ func TestConfigurationLabelValidation(t *testing.T) {
 			Spec: validConfigSpec,
 		},
 		want: apis.ErrMissingField("metadata.labels.serving.knative.dev/service"),
-	}, {
```

Comment:
should we add a test to make sure that we DON'T reject unknown uses of our prefix?

Reply:
I don't think it's necessary to have a negative test case

Reply:
To me it’s not so much a negative test as a test to ensure we can add more labels without something breaking.